src/Controller/EnMarche/Coordinator/CoordinatorCitizenProjectController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche\Coordinator;
+
+use AppBundle\Coordinator\Filter\AbstractCoordinatorAreaFilter;
+use AppBundle\Coordinator\Filter\CitizenProjectFilter;
+use AppBundle\Entity\CitizenProject;
+use AppBundle\Exception\BaseGroupException;
+use AppBundle\Form\CoordinatorAreaType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/**
+ * @Route("/espace-coordinateur/projet-citoyen")
+ * @Security("has_role('ROLE_COORDINATOR_CITIZEN_PROJECT')")
```

Comment:
yes, like any other role, but it wasn't planned :)

Reply:
It's important to know, if it can be then `is_granted` should be used instead of `has_role` to trigger the hierarchy voter and not rely on a simple token check.

Reply:
Actually, neither the role `ROLE_COORDINATOR_COMMITTEE` nor the role `ROLE_COORDINATOR_CITIZEN_PROJECT` can be inherited.

Although i see no valid reason to say they won't be inherited someday.

Let's go for `is_granted` ?

Reply:
I would say so.